src/WebService/Client.php

```diff
@@ -277,7 +285,7 @@ private function jsonErrorDescription(): string
      */
     private function urlFor(string $path): string
     {
-        return 'https://' . $this->host . $path;
+        return $this->protocol . $this->host . $path;
```

Comment:
I used the built-in php web server for the tests. However, the built-in php web server does not handle https requests. I was either going to have to create a tunnel to the server using something like `stunnel3` or enable http like above. Would that cause a problem?